### PR TITLE
Add pt locale in v2

### PR DESCRIPTION
**Pull request by adrm**
Closes https://github.com/date-fns/date-fns/issues/1012

### CODE REVIEW

File: src/locale/pt/test.js [hunk 1]
```diff
@@ -1,9 +1,493 @@
...
+          new Date(1986, 3, 4, 10, 32, 0),
+          new Date(1986, 3, 4, 11, 32, 0),
+          { locale: locale, addSuffix: true }
+        )
+        assert(result === 'há 1 hora')
+      })
+    })
+  })
+
+  context.skip('with `formatRelative`', function() {
```
Comment (kossnocorp) [inline src/locale/pt/test.js]: Please remove `.skip`

Comment (adrm) [reply src/locale/pt/test.js]: Removed. Thanks for noticing it and sorry for letting it slip through.

**Comment by adrm** [comment]
@kbrabrand The problems you pointed out are fixed and all tests are passing.

**Comment by kossnocorp** [comment]
Thanks!

**Pull request merged**